doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nRF54H20-1.rst

```diff
@@ -10,7 +10,7 @@
 |NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
 The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
 
-The |NCS| v2.9.0-nRF54H20-1-rc3 is an nRF54H20-exclusive release tag, only supporting products based on the nRF54H20 SoC.
+The |NCS| v2.9.0-nRF54H20-1 is an nRF54H20-exclusive release tag, only supporting products based on the nRF54H20 SoC.
```

Comment:
The comment is about line 53 (NCSDK-30802), but cannot comment there as it wasn't changed. Please remove `after nrfxlib 3.9.0.` from that line, or change to ` after nrfx 3.9.0.` The issue occurred when nrfx 3.9.0 was integrated in upstream, that's why the task refers to it.

Reply:
Additionally, shall this statement be also present in the `Enhanced ShockBurst (ESB)` section, or would it be a duplication?

Reply:
Fixed and added under ESB